packages/react-dom/src/__tests__/ReactDOMFiber-test.js

```diff
@@ -1052,7 +1046,11 @@ describe('ReactDOMFiber', () => {
     class Click extends React.Component {
       constructor() {
         super();
-        node.click();
+        expect(() => {
+          node.click();
+        }).toWarnDev(
+          'Warning: unstable_flushDiscreteUpdates: Cannot flush updates when React is already rendering.',
```

Comment:
Should bikeshed this before it hits open source

Reply:
I'll leave you the key to the bikeshed. Help yourself.